frontend/sampledata/basic/personnel.txt

```diff
@@ -0,0 +1,2 @@
+firstname	lastname	role
+John	Bithorn	Field Technician
```

Comment:
Sample data only be checked in if it is fake data. Real data should never be in our git history.

Reply:
deleted from remote and added to gitignore!